src/python/grpcio/grpc/_server.py

```diff
@@ -386,10 +386,18 @@ def unary_request():
     return unary_request
 
 
-def _call_behavior(rpc_event, state, behavior, argument, request_deserializer):
+def _call_behavior(rpc_event,
+                   state,
+                   behavior,
+                   argument,
+                   request_deserializer,
+                   stream_observer=None):
     context = _Context(rpc_event, state, request_deserializer)
     try:
-        return behavior(argument, context), True
+        if stream_observer is not None:
+            return behavior(argument, context, stream_observer), True
```

Comment:
Should we use keyword argument `on_next = stream_observer`?

Reply:
I'm not sure. Leaving the keyword argument out makes it easier to change to something more readable in the future, for what it's worth.